src/Button/Button.tsx

```diff
@@ -42,64 +38,45 @@ export const Button: FC<ButtonProps> = forwardRef<
     children,
     id,
     className = '',
-    color = 'liquorice',
-    block = false,
-    inverted = false,
     disabled = false,
-    outlined = false,
     handleClick,
     loading = false,
     primary = false,
     secondary = false,
     tertiary = false,
+    fallback = false,
+    textBtn = false,
     icon = '',
+    trailingIcon = false,
     forcedWidth = '',
     form,
     type,
     ...otherProps
   } = props
 
-  const isLegacyButton = !primary && !secondary && !tertiary
-
   useDeprecatedWarning({
-    enabled: isLegacyButton,
-    title: 'Legacy Button',
+    enabled: tertiary,
+    title: 'Tertiary prop usage',
     message:
-      "You're using the legacy Button component. Please use the new Button by providing one of the following props: 'primary', 'secondary', 'tertiary'.",
+      "You're using the tertiary prop which is now deprecated. Please use the new 'fallback' prop instead, or 'primary' or 'secondary' as appropriate.",
     componentProps: props,
   })
```

Comment:
If `tertiary` and `fallback` are similar or `fallback` is replacing it, we could just remove `tertiary` entirely.